README.md

```diff
@@ -31,7 +31,7 @@ You can learn about the proxy pattern and how to use the **ProxyManager** in the
 The suggested installation method is via [composer](https://getcomposer.org/):
 
 ```sh
-php composer.phar require ocramius/proxy-manager:1.0.*
+php composer.phar require ocramius/proxy-manager:~2.0
```

Comment:
We should just drop the version. composer now automatically resolves to the latest stable version :-)